Makefile

```diff
@@ -81,6 +81,9 @@ helm: ## Generate Helm Chart
 website: ## Generate Docs Website
 	cd website;npm install;hugo -d ../docs
 
+website: ## Generate Docs Website
```

Comment:
Is there a difference between this target and the one above? Will doing `make website` hit both of these targets? Can we collapse this into one?

Reply:
Ahh apologies, bad merge resolution. I'll fix